app/views/watchlists/show.html.erb

```diff
@@ -47,7 +47,7 @@
   <% unless params[:all] %>
     <div class="row my-2">
       <div class="col-12">
-        <%= paginate @watchlist_entries, window: 2 %>
```

Comment:
Other media searches also have this navbar on top, not only at the bottom of the page.